ci-operator/step-registry/ipi/conf/aws/blackholenetwork/blackhole_vpc_yaml.md

```diff
@@ -173,6 +173,25 @@ Resources:
     Properties:
       SubnetId: !Ref PrivateSubnet3
       RouteTableId: !Ref PrivateRouteTable3
+  EC2Endpoint:
```

Comment:
Should a TODO comment be added to remove the this VPCE when https://bugzilla.redhat.com/show_bug.cgi?id=1769223 is fixed?

Reply:
I dunno.  We will probably need to keep this until we have end-of-lifed all releases for which the machine-API needs the endpoint.  But we may want to split into two flavors of VPC so we can test proxy-respecting machine-API while continuing to run CI on proxy-ignoring machine-API.  I'm fine tracking that in Jira or a bug or whatever instead of in release comments.